.circleci/config.yml

```diff
@@ -28,4 +36,33 @@ jobs:
             docker cp ./kitsune/search/dictionaries/synonyms/. project_elasticsearch7_1:/usr/share/elasticsearch/config/synonyms
       - run:
           name: Run unit tests
-          command: make test-ci
+          command: ./bin/dc.sh run test ./bin/run-unit-tests.sh
+      - run:
+          name: Run js tests
+          command: ./bin/dc.sh run test ./bin/run-mocha-tests.sh
+      - when:
+          condition:
+            equal: [ main, << pipeline.git.branch >> ]
```

Comment:
> how about splitting the jobs a bit more (lint, test-js, unittests, build). 

This was my initial approach, however I couldn't immediately figure out how to share the docker context between each job, and so each had to build the image again completely from scratch - which took rather a long time. So I opted for the simpler (faster) approach first, to avoid this PR getting too unwieldily.

But my thought was, once merged, we can explore a bit more of the features Circle has to offer - like "proper" workflows, pretty test result output, and parallel tests.